### PR TITLE
Renamed RcDirs->RcManager

### PR DESCRIPTION
**Pull request by mstefferson**
# Context

I found `util.py` not a helpful module name nor `RcDirs` an intuitive class name. So I renamed

# Changes

* Rename `util.py` -> `rcmanager.py`
* Renamed `RcDirs`
* renamed `rc_dirs` and `rcdirs`

# Behaves Differently

* Nothing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,7 +26,7 @@ ci: ci-black ci-flake8 ci-test ## Check black, flake8, and run unit tests
 	@echo "CI sucessful"
 
 isort: build-docker  ## Runs isort to sorts imports
-	docker run -v ${PWD}:/mnt -w /mnt -it $(IMAGE) isort -rc $(PROJ_DIR)
+	docker run -v ${PWD}:/mnt -w /mnt -it $(IMAGE) isort $(PROJ_DIR)
```
Comment (mstefferson) [inline Makefile:29]: This is unrelated. New version of `isort` doesn't require this flag. This removes an `isort` warning.

**Review by jeffmm — APPROVED** [review]
lgtm
